src/pages/instances/Events.tsx

```diff
@@ -7,19 +7,46 @@ import { useQueryClient } from "@tanstack/react-query";
 import { queryKeys } from "util/queryKeys";
 import { useOperations } from "context/operationsProvider";
 import { useNotify } from "@canonical/react-components";
+import useEventListener from "util/useEventListener";
 
 const EVENT_HANDLER_DELAY = 250;
 const WS_RETRY_DELAY_MULTIPLIER = 250;
 const MAX_WS_CONNECTION_RETRIES = 5;
+const EVENT_WS_STALE_TIMEOUT_MS = 3600_000;
```

Comment:
If we want to use the underscore separator correctly I think this should be:
```suggestion
const EVENT_WS_STALE_TIMEOUT_MS = 3_600_000;
```